### PR TITLE
Introduce a Random Option and UI improvements

### PR DESCRIPTION
**Pull request by etothepii**
Improved the Frequency counter so it can go to infinity.
Added a random option so that the shuffling of dice options can be random or can happen at a chosen point.
Improved the UI a little to fit better on iPhone X and IPad Pro 2

### CODE REVIEW

File: boardgame dice/FrequencyCounter.swift [hunk 1]
```diff
@@ -34,7 +34,7 @@ struct FrequencyCounter_Previews: PreviewProvider {
         stateModel.setRoll(roll: 4,value: 3)
         stateModel.setRoll(roll: 5,value: 4)
         stateModel.setRoll(roll: 6,value: 5)
-        stateModel.setRoll(roll: 7,value: 6)
+        stateModel.setRoll(roll: 7,value: 18)
```
Comment (mgh35) [inline boardgame dice/FrequencyCounter.swift:37]: Is this intended? This is a test object?

File: boardgame dice/FrequencyCount.swift [hunk 2]
```diff
@@ -8,23 +8,37 @@
...
+        }
+        return count % 8
+    }
+    
     var body: some View {
-        VStack {
-            Text(LEFT_BRAILLE[count]).scaledFont(name: "Georgia", size: 24)
+        let braile = roll > 7 ? LEFT_BRAILLE : RIGHT_BRAILLE
+        VStack(alignment: .center) {
+            ForEach((0...max(1,count/8)).reversed(), id: \.self) { row in
```
Comment (mgh35) [inline boardgame dice/FrequencyCount.swift:32]: It was hard to figure out what this is doing.

This is the bit updating the counters so they can count to infinity? So it's figuring out the number of full multiples of 8 and then the remainder as a partial one?

I wonder if it's clearer to express along the lines of (in Python for lack of understanding Swift):

```python
def make_count_charcter_indices(count: int) -> List[int]:
  indices = []
  while count > 8:
    indices.append(8)
    count -= 8
  indices.append(count)
  return indices
```

and then iterator over those indices to render it.

**Review by mgh35 — APPROVED** [review]
I'm struggling to understand the UI syntax, but sounds like that's been well beta-tested.

I note there are no new tests? :)

File: boardgame dice/FrequencyCounter.swift (hunk 1 above)
Comment (etothepii) [reply boardgame dice/FrequencyCounter.swift:37]: These sort of show what what the view would look like with this state, so now being able to show numbers more than 6 I wanted to see that it looked right.

**Pull request merged**